storage/src/vespa/storage/visiting/visitor.cpp

```diff
@@ -192,6 +192,8 @@ Visitor::Visitor(StorageComponent& component)
       _hitCounter(),
       _trace(DEFAULT_TRACE_MEMORY_LIMIT),
       _messageHandler(nullptr),
+      _messageSession(),
+      _documentPriority(documentapi::Priority::PRI_NORMAL_1),
```

Comment:
Consider setting `PRI_NORMAL_3` instead as that corresponds to the default client feed priority (yes I know, it's not exactly obvious and one more reason to throw onto the "remove client operation priorities" pile 🥸).